docs/source/examples/managed-jobs.rst

```diff
@@ -112,7 +113,7 @@ To launch this YAML as a managed job, use :code:`sky jobs launch`:
 
   $ sky jobs launch -n bert-qa-job bert_qa.yaml
 
-:code:`sky jobs launch` works similarly to :code:`sky launch` - you can run :code:`sky jobs launch --help` or see the :ref:`CLI reference <sky-job-launch>` for more information.
+:code:`sky jobs launch` has a similar interface to :code:`sky launch` - you can run :code:`sky jobs launch --help` or see the :ref:`CLI reference <sky-job-launch>` for more information.
```

Comment:
Still not sure about keeping this para. Some major arguments are missing in `jobs launch`, so this statement isn't 100% true. Still repeats a para below.

Reply:
ok will just delete it